src/components/loader/loader-square.test.tsx

```diff
@@ -32,7 +32,7 @@ it.each([0, 1, 2])("each loader square renders as expected", (index) => {
 
 // These styling tests required for coverage
 test("when `size` prop is set to large, the expected width, height, and margin styles are applied", () => {
-  mockUseMediaQuery.mockReturnValueOnce(true);
+  mockUseMediaQuery.mockReturnValueOnce(false);
```

Comment:
comment: the should be `true`, but as there is a `beforeEach` at the top of the file already you don't need to have any mock value here